### ISSUE TITLE
About pre-training on the ava dataset

### ISSUE THREAD

**Issue by Deep-learning999** [facebookresearch/TimeSformer#1]
Excuse me, there is pre-training of the ava data set. If you want to train and infer on the ava data set, how should you proceed? If you have trained on the ava data set, is there a parameter chart that compares with slowfast? How many fps can be achieved

**Comment by gberta** [comment]
We haven't done any experiments on the AVA dataset. To do this, you would need to modify TimeSformer architecture so that it could also predict bounding boxes in addition to the action labels.

**Comment by Deep-learning999** [comment]
According to your estimation, if you train on ava, you can detect multi-person actions, so that the speed will exceed slowfast

**Comment by gberta** [comment]
I don't have much experience with training SlowFast on AVA so I can't give a very detailed answer. If you train SlowFast from scratch, then TimeSformer will certainly be faster. However, if you finetune SlowFast from Kinetics, I'm not sure what the training cost comparison between the two would be. In terms of inference, I would say that TimeSformer is faster than SlowFast as it requires fewer clips for testing.

**Comment by chenqqqqqqq** [comment]
> 根据你的估计，如果你在ava上训练，你可以检测到多人的动作，这样速度就会超过慢快

Have you implemented TimeSformer's training on the ava dataset？

**Comment by chenqqqqqqq** [comment]
> I don't have much experience with training SlowFast on AVA so I can't give a very detailed answer. If you train SlowFast from scratch, then TimeSformer will certainly be faster. However, if you finetune SlowFast from Kinetics, I'm not sure what the training cost comparison between the two would be. In terms of inference, I would say that TimeSformer is faster than SlowFast as it requires fewer clips for testing.

Hello, Have you implemented TimeSformer's training on the ava dataset？
